### DISCUSSION TITLE
Logging connections stopped working - is it an API problem?

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#2559] [General]
Logging connections stopped working - is it an API problem?
Conducted connections are not logged using the GridTracker program
There is an error in its interface, I can’t understand why it appeared?
I made a new key and it doesn't work either
Log version 2.4.10
Where can I find a fix for this error?

**Comment by MikleBas** [comment]
UPD - server error log

```
IP-server - - [10/Oct/2023:21:28:36 +0300] ＂POST / HTTP/1.1＂ 303 5 ＂-＂ ＂GridTracker/1.23.0402＂
IP-server - - [10/Oct/2023:21:28:37 +0300] ＂POST / HTTP/1.1＂ 303 5 ＂-＂ ＂GridTracker/1.23.0402＂
IP-server - - [10/Oct/2023:21:28:43 +0300] ＂POST / HTTP/1.1＂ 303 5 ＂-＂ ＂GridTracker/1.23.0402＂
```

**Reply by magicbug** [reply]
I'd suggest asking grid tracker developers nothing in cloudlog has changed that should break apps using the api

**Reply by MikleBas** [reply]
the tracker hasn’t been updated for a long time, what version was it remains the same
